plugins/bundle/plugin.go

```diff
@@ -47,10 +48,12 @@ type Plugin struct {
 // New returns a new Plugin with the given config.
 func New(parsedConfig *Config, manager *plugins.Manager) *Plugin {
 	initialStatus := map[string]*Status{}
+	loggers := make(map[string]sdk.Logger, len(parsedConfig.Bundles))
```

Comment:
Can we avoid instantiating multiple loggers here? We're going to run into issues if the plugin gets reconfigured to download different bundles. We could address that by instantiating new loggers on reconfiguration, but I don't think that's worth it.

A single logger that allows a set of key-value pairs to be passed to the log entry would be enough here.

Reply:
Fixed.